### DISCUSSION TITLE
PGN 127493 - Transmission Parameters, Dynamic

### DISCUSSION THREAD

**Discussion by PhiLehr** [canboat/canboat#419] [General]
Dear N2K Fellows, 
not a question or discussion, but additional information for the Wiki

Could not find an other thread to post it in so if you tell me where I can share my findings in the future I will share them there then.

In 0x1F205: PGN 127493 - Transmission Parameters, Dynamic
Field #6 Discrete Status 1 (0-255):
Bit	Fault
0	Check transmission
1	Over Temperature
2	Low oil pressure
3	Low oil level
4	-
5	-
6	-
7	-

Reverse engineered it on Raymarine MFD with newest software
Hope it helps

**Comment by sbender9** [comment]
4 is "Sail Drive", the rest are "reserved"

It's pretty simple to add these to pgn.h/lookup.h

**Comment by keesverruijt** [comment]
This is fixed in #667
